debian.aws/config/OVERRIDES

```diff
@@ -5,14 +5,17 @@ CONFIG_6LOWPAN=n
 CONFIG_6LOWPAN_NHC=n
 CONFIG_BT=n
 CONFIG_CAIF=n
+CONFIG_CFG80211=n
```

Comment:
I think if you include `CONFIG_WIRELESS=n`, then that omits the entire `net/wireless` module, and you won't need anything else related to 80211.

Reply:
OK, I can remove it.